docs/Contributors-Guide/Bug-Triage.md

```diff
@@ -272,31 +272,29 @@ During triaging you might come across a particular bug which is present
 across multiple version of GlusterFS. Here are the course of actions:
 
 -   We should have separate bugs for each release (We should
-    clone bugs if required)
--   Bugs in released versions should be depended on bug for mainline
-    (master branch) if the bug is applicable for mainline.
-    -   This will make sure that the fix would get merged in master
-        branch first then the fix can get ported to other stable
-        releases.
+    clone bugs if required), the bug needs to be fixed.
+
+-   Bug reported for any specific version neednot be cloned for
+    'mainline', as by default, as a policy, we need to submit the fix
+    for 'master' branch before backporting fix to specific branch.
+    Only when a bug needs to be fixed in multiple 'released' versions,
+    then there is a clone requried.
```

Comment:
This is not particularly clear to me. We have users reporting minor bugs against an old, but still maintained version (say 3.12). Does this mean the bug needs to get fixed in 'mainline', and 3.12 only? The reporter may not care about a fix for 4.1. I guess it would need a clone for 4.1 too as we require fixes to be backported to all newer maintained releases too. However we now have the clone for 4.1 that depends on the original 3.12 bug that has the mainline fix. This looks confusing to me.

I don't think we can say 'send a patch for mainline against a bug reported against the most recent maintained version', as some patches linger around waiting for reviews while a new version gets branched.

In the end, I think it makes it easier to miss backports to maintained/branched releases. The release scripts that verified backports, parents, related BZs and their status is non-functional for a while already (not even sure where it is burried now). So, maybe our use of mixing Bugzilla+GitHub for tracking allows us to be less strict on the Bugzilla workflow too.

Also, replace **'released' versions** with **maintained versions**.

Reply:
Done